code/game/objects/items/storage/firstaid.dm

```diff
@@ -107,11 +106,11 @@
 		/obj/item/stack/medical/bruise_pack = 2,
 		/obj/item/stack/medical/ointment = 2,
 		/obj/item/reagent_containers/hypospray/medipen = 1,
-		/obj/item/healthanalyzer = 1,
 		/obj/item/surgical_drapes = 1,
 		/obj/item/scalpel = 1,
 		/obj/item/hemostat = 1,
-		/obj/item/cautery = 1)
+		/obj/item/cautery = 1,
+		/obj/item/healthanalyzer = 1)

```

Comment:
?

Reply:
Messed up while writing this, the analyzer got removed in 110.
This is where MDs get their roundstart scanners from.

Reply:
Think it would be good to just put it as part of the doctor arsenal instead of in the kit?

There's a few in medivend (big) and the health scanning is a function within the PDA cartridge doctors spawn with (although it was never given a chemical scan iirc).

On box I know there's a couple of whities with dedicated spawns (gateway, dorms).